### PR TITLE
Shiny new “warning” flag.

### PR DESCRIPTION
**Pull request by Wilto**
Veeery low priority change. Look okay to everyone?

### CODE REVIEW

**Comment by Wilto** [comment]
image

**Comment by anselmh** [comment]
LGTM! :)

File: index.bs [hunk 1]
```diff
@@ -27,18 +27,59 @@ Link Defaults: html5 (element) source/video/audio/iframe
 [data-link-type=element]::before { content: "<" }
 [data-link-type=element]::after { content: ">" }
 
-#obsolete { position:fixed; z-index:1; background:#EEFFEE; border:solid #99FF99 0.25em; padding:1em; top:40%; left:20%; right:20% }
-#obsolete ~ *, #obsolete ~ * * { color:gray !important; background:transparent !important }
```
Comment (zcorpan) [inline index.bs:31]: Can you keep this line?

Comment (Wilto) [inline index.bs:31]: Did a little `opacity` biz, which seems a little more consistent. This work, @zcorpan?

image

Comment (zcorpan) [inline index.bs:31]: Yeah sure

**Pull request merged**